lib/reports/ReportSpeedPercentile.js

```diff
@@ -504,7 +504,9 @@ class ReportSpeedPercentile extends ReportBaseFlow {
       const tableOptions = ReportSpeedPercentile.getTableOptions(reportBlock);
       layout.push({ type: ReportBlock.METADATA, options: countMetadataOptions });
       layout.push({ type: ReportBlock.TABLE, options: tableOptions });
+      layout.push({ type: ReportBlock.PAGE_BREAK, options: {} });
     });
+    layout.pop();
```

Comment:
Presumably to remove the extra blank page that is created?

Reply:
Yep

Reply:
cool beans!